demos/smartwatch/lv_demo_smartwatch_notifications.c

```diff
@@ -0,0 +1,344 @@
+/**
```

Comment:
Could you add more realistic sample messages?

Reply:
I have updated it. As for the tileview I was considering using it but since the original code did not have this, it would take longer to adapt it to the existing but we can still update it later. Definitely, making it support up to 400x400 is in the plan